definition.tex

```diff
@@ -53,6 +53,8 @@ \section{Defining Proof-of-Burn}
 
 \import{./}{algorithms/alg.bind-game.tex}
 
+It is desired for the protocol that a burn address generated encodes one and only one tag, so that an adversary cannot create an address that commits to multiple things. Specifically, if we fix a burn address $\burnAddr$, then $\BurnVerify(1^\kappa, t, \burnAddr)$ should only evaluate to $\true$ for a single tag $t$. To state this property we introduce another game-based security definition. The binding game $\bindattack$ is illustrated in Algorithm~\ref{alg.bind-game}.
```

Comment:
Shorten:

* Remove "for the protocol"
* remove "generated"
* remove ", so that an adversary cannot create an address that commits to multiple things"
* Specifically -> Concretely
* if we fix -> given
* remove "then"
* "To state this property we introduce another game-based security definition. The binding game $\bindattack$ is illustrated in Algorithm~\ref{alg.bind-game}" -> "The game $\bindattack$ in Algorithm~\ref{alg.bind-game} captures this property."